src/components/AqiGauge/AqiDial.tsx

```diff
@@ -18,9 +19,9 @@ interface DialProps {
  * This component displays the label for the AQI displayed by the dial.
  * @example if the AQI is less than 50, it will show the label "Good"
```

Comment:
This is from an old PR, but example is meant for code examples, so code editors render this line as code.
```suggestion
 * For example, if the AQI is less than 50, it will show the label "Good"
```